keystone/identity/backends/hybrid.py

```diff
@@ -19,7 +19,7 @@
 LDAP_USER_ID_ATTRIBUTE = CONF.ldap.user_id_attribute
 LDAP_USER_TREE_DN = CONF.ldap.user_tree_dn
 LDAP_GENERIC_TREE_DN = CONF.ldap.generic_tree_dn
-LDAP_BUILTIN_USERS = CONF.ldap.builtin_users
+LDAP_BUILTIN_USERS = CONF.ldap.builtin_users.split(',')
```

Comment:
Any chance this would be a type other than a string? This would result in a syntax error if say CONF.ldap.builtin_users a number and an attribute error if CONF.ldap.builtin_users was None. I am unsure how the keystone config behaves so it may do sane defaults for example if CONF.ldap.builtin_users was not specified in the config.